pulp_rpm/app/migrations/0061_fix_modulemd_defaults_digest.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.10 on 2024-04-15 18:44
+
+import hashlib
+
+from django.db import migrations
+
+
+def add_snippet_hash(apps, schema_editor):
+    """Calculate and add digest hash of the snippet."""
+
+    Modulemd = apps.get_model("rpm", "ModulemdDefaults")
+    modules_to_update = []
+
+    for mmd in Modulemd.objects.filter(digest__in=('', None)).only("snippet").iterator():
+        mmd.digest = hashlib.sha256(mmd.snippet.encode()).hexdigest()
+        modules_to_update.append(mmd)
+    Modulemd.objects.bulk_update(modules_to_update, fields=["digest"])
```

Comment:
I've used this as a basis: https://github.com/pulp/pulp_rpm/blob/main/pulp_rpm/app/migrations/0052_modulemd_digest.py

It worked with a simple manual testing.

Reply:
+1